ubi9-py39/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Thoth's extension to OpenShift's S2I build
+FROM registry.access.redhat.com/ubi9/python-39:1-59
+
+ENV SUMMARY="Thoth's Source-to-Image for Python ${PYTHON_VERSION} applications" \
+    DESCRIPTION="Thoth's Source-to-Image for Python ${PYTHON_VERSION} applications. This toolchain is based on Red Hat UBI9. It includes pipenv." \
+    THOTH_S2I_NAME=quay.io/thoth-station/s2i-thoth-ubi9-py39 \
+    THOTH_S2I_VERSION=0.34.1 \
```

Comment:
@harshad16 I wonder if we should remove these labels?!

Reply:
These env vars are used down the line , for thoth resolver notice which version of thoth base image were used.

Reply:
@Gregory-Pereira  can you update this version to `v0.35.0`